core/templates/dev/head/editor/feedback_tab/ThreadDataService.js

```diff
@@ -167,8 +167,8 @@ oppia.factory('threadDataService', [
         payload.commit_message = commitMsg;
       }
       _openThreadsCount -= 1;
-      $http.put(_SUGGESTION_ACTION_HANDLER_URL + threadId, payload).success(
-        onSuccess).error(function() {
+      $http.put(_SUGGESTION_ACTION_HANDLER_URL + threadId, payload).then(
+        onSuccess, function() {
```

Comment:
Is onSuccess expecting a response object or a data object? Please check, and test manually.